src/media/js/error_messages.js

```diff
@@ -0,0 +1,26 @@
+/*
+    Error codes and their associated messages.
+*/
+define('error_messages', ['core/l10n'], function(l10n) {
+    'use strict';
+    var gettext = l10n.gettext;
+
+    return {
```

Comment:
Yeah but we don't want special l10n for every page. It increases the file size for each locale and introduces more l10n work. If you want to make it specialized, pass a noun into gettext and format it in. But it'd be good for the keys to be 403/404/500, etc using response.status_code